src/test/java/com/hortonworks/spark/sql/hive/llap/MockHiveWarehouseSessionImpl.java

```diff
@@ -36,27 +38,27 @@ static DriverResultSet testFixture() {
         return new DriverResultSet(row, schema);
     }
 
-    MockHiveWarehouseSessionImpl(HiveWarehouseSessionState sessionState) {
+    public MockHiveWarehouseSessionImpl(HiveWarehouseSessionState sessionState) {
         super(sessionState);
         super.getConnector =
                 () -> new MockConnection();
         super.executeStmt =
                 (conn, database, sql) -> {
                     try {
-                        org.apache.hadoop.hive.ql.parse.ParseUtils.parse(sql);
+                        new org.apache.hadoop.hive.ql.parse.ParseDriver().parse(sql);
                         return testFixture();
                     } catch(ParseException pe) {
                         throw new RuntimeException(pe);
                     }
                 };
       super.executeUpdate =
         (conn, database, sql) -> {
-          try {
-            org.apache.hadoop.hive.ql.parse.ParseUtils.parse(sql);
-            return true;
-          } catch(ParseException pe) {
-            throw new RuntimeException(pe);
-          }
+            try {
+                new org.apache.hadoop.hive.ql.parse.ParseDriver().parse(sql);
```

Comment:
`ParseDriver().parse()` existent in both Spark's hive and hive 3.0.0 too. Seems safer to use it.